packages/core/src/App/Containers/RealAccountSignup/helpers/utils.js

```diff
@@ -18,6 +18,11 @@ function splitValidationResultTypes(values) {
         }
     });
     return { warnings, errors };
-}
+};
 
-export { splitValidationResultTypes };
+export const validate = (errors, values) => (fn, arr, err_msg) => {
```

Comment:
Is it possible to use existing `Validator`(referring to `validator.js`)

Reply:
@balakrishna-binary, as I checked these, are two different functions for validation.